frontend/test/admin/people/people.integ.spec.js

```diff
@@ -22,15 +22,21 @@ import UserGroupSelect from "metabase/admin/people/components/UserGroupSelect";
 import { GroupOption } from "metabase/admin/people/components/GroupSelect";
 import { UserApi } from "metabase/services";
 import UserActionsSelect from "metabase/admin/people/components/UserActionsSelect";
+import MailDev from "maildev"
 
 describe("admin/people", () => {
   let createdUserId = null;
 
   beforeAll(async () => {
     useSharedAdminLogin();
+
   });
 
   describe("user management", () => {
+
+    let maildev = MailDev({smtp: 4025});
```

Comment:
It would probably be best to do

```javascript
const maildev = MailDev({smtp: 4025});
beforeAll(() =>
  new Promise((resolve, reject) => maildev.listen(err => err ? reject(err) : resolve())
)
afterAll(() =>
  new Promise((resolve, reject) => maildev.close(err => err ? reject(err) : resolve()))
)
```

(yay, Node callbacks ಠ_ಠ)

This will ensure it's started and stopped properly.